legate/core/runtime.py

```diff
@@ -1543,6 +1539,193 @@ def create_store(
             ndim=ndim,
         )
 
+    def create_manual_task(
+        self,
+        context: Context,
+        task_id: int,
+        launch_domain: Rect,
+    ) -> ManualTask:
+        """
+        Creates a manual task.
+
+        Parameters
+        ----------
+        context: Context
+            Library to which the task id belongs
+
+        task_id : int
+            Task id. Scoped locally within the context; i.e., different
+            libraries can use the same task id. There must be a task
+            implementation corresponding to the task id.
+
+        launch_domain : Rect, optional
+            Launch domain of the task.
+
+        Returns
+        -------
+        ManualTask
+            A new task
+        """
+
+        from .operation import ManualTask
```

Comment:
This import no longer needed. Similar for other imports from `.operation` in functions copied from context.py.

Reply:
Why is that unnecessary? Note that the import for `ManualTask` at the beginning of this file is only for type checking.

Reply:
fyi, if I remove the function local imports, I see name errors.

Reply:
OK, I thought those imports could be moved to the top level, but that's not possible because it would cause a circular import.